src/settings-ui/Settings.UI/ViewModels/MouseWithoutBordersViewModel.cs

```diff
@@ -736,6 +736,28 @@ public bool ValidateRemoteMachineIP
             }
         }
 
+        public string Name2IP
+        {
+            // Due to https://github.com/microsoft/microsoft-ui-xaml/issues/1826, we must
+            // add back \n chars on set and remove them on get for the widget
+            // to make its behavior consistent.
```

Comment:
"its behavior consistent." -> "its behavior consistent with the old UI and MWB internal code."